test/e2e/simple_test.go

```diff
@@ -82,6 +83,7 @@ func TestSimpleBuild(t *testing.T) {
 			Name:      buildName,
 		},
 		Spec: v1alpha1.BuildSpec{
+			Timeout: "40s",
```

Comment:
Can we use an actual `time.Duration` for this field?

Reply:
Thats a great time. I have updated the PR with this change.